core/store/models/ocrkey/key_bundle.go

```diff
@@ -56,53 +52,10 @@ type (
 )
 
 var (
-	ErrScalarTooBig = errors.Errorf("can't handle scalars greater than %d", curve25519.PointSize)
+	errScalarTooBig = errors.Errorf("can't handle scalars greater than %d", curve25519.PointSize)
```

Comment:
It's probably better to leave this exported so that another package can make use of:
```go
if errors.Cause(err) == ErrScalarTooBig {
```